app/views/admin/home/show.html.erb

```diff
@@ -1,3 +1,41 @@
-<h1>Admin page</h1>
+<h1 class="govuk-heading-xl">Overview</h1>
 
-<p>Not sure if we'll need it yet but here's a admin overview page</p>
+<table class="govuk-table" style="width: auto;">
+  <caption class="govuk-table__caption govuk-table__caption--m">Schools registered for 2023 to 2024</caption>
+  <thead class="govuk-table__head">
+    <tr class="govuk-table__row">
+      <th scope="col" class="govuk-table__header">Training programme</th>
+      <th scope="col" class="govuk-table__header govuk-table__header--numeric">Schools</th>
+    </tr>
+  </thead>
+  <tbody class="govuk-table__body">
+    <tr class="govuk-table__row">
+      <th scope="row" class="govuk-table__header govuk-!-font-weight-regular">Delivering their training using DfE materials</th>
+      <td class="govuk-table__cell govuk-table__cell--numeric"><%= @pilot_stats.cip_total %></td>
+    </tr>
+    <tr class="govuk-table__row">
+      <th scope="row" class="govuk-table__header govuk-!-font-weight-regular">Designing their own programme</th>
+      <td class="govuk-table__cell govuk-table__cell--numeric"><%= @pilot_stats.diy_total %></td>
+    </tr>
+    <tr class="govuk-table__row">
+      <th scope="row" class="govuk-table__header govuk-!-font-weight-regular">Using a training provider (full induction programme)</th>
+      <td class="govuk-table__cell govuk-table__cell--numeric govuk-!-font-weight-regular"><%= @pilot_stats.fip_total %></td>
+    </tr>
+    <% @pilot_stats.partnership_totals.each do |provider| %>
```

Comment:
If you liked my suggestion above, this would need to be changed as well to something like this:
`<% @pilot_stats.partnership_totals.each do |(provider_name, provider_total)| %>`